data/siggraph-2019/10.1145-3306346.3323017.json

```diff
@@ -0,0 +1,203 @@
+[
+    {
+        "##": "Fields with prefix ## are just comments to clarify entries",
+
+        "##": "Name of the variant. This can be used to distinguish",
+        "##": "between different OS or build environments. Common variant names should be formatted as",
+        "##": "Ubuntu xx.xx, Debian xx.xx, Windows xx.xx, MacOS x.",
+        "Variant name": "linux (ubuntu)",
+
+        "##": "Reviewer name and email",
+        "##": "E.g. 'John Doe <john.doe@world.org>'",
+        "##": "Note: we will detect and add a link to your github account if you use the same email address here.",
+        "Reviewer name": "David Coeurjolly <david.coeurjolly@liris.cnrs.fr>",
+
+        "##": "For a given paper, there is one specific variant (the master one)",
+        "##": "that is displayed on the first pane of the replicability paper web page",
+        "##": "(true/false)",
+        "Is master variant (boolean)": true,
+        
+        "##": "Some variants can be deprecated (and are thus not displayed)",
+        "##": "For new contributions, this should be set to 'false'",
+        "##": "(true/false)",
+        "Is variant deprecated (boolean)": false,
+
+        "##": "Article title",
+        "Title": "Finding hexahedrizations for small quadrangulations of the sphere",
+
+        "##": "Article identifiers",
+        "DOI": "10.1145/3306346.3323017",
+
+        "##": "Year of publication (integer)",
+        "Year": 2019,
+
+        "##": "ACM Keywords of the paper (a list of strings)",
+        "ACM Keywords": [
+            "Mesh geometry models",
+            "Combinatorial optimization",
+            "Permutations and combinations"
+        ],
+
+        "##": "Paper main topic",
+        "##": "Please refer to the original article of the project for details on this",
+        "##": "topic classification.",
+        "Topic {Rendering, Animation and Simulation, Geometry, Images, Virtual Reality, Fabrication}": "Geometry",
+
+        "##": "Is any of the co-authors from a university of a public institution ? (true/false)",
+        "Co-authors from academia (boolean)": true,
+
+        "##": "Is any of the co-authors from a private company? (true/false)",
+        "Co-authors from industry (boolean)": false,
+
+        "##": "Is the paper available as ACM Open Access? (true/false) ",
+        "ACM Open Access (boolean)": false,
+
+        "##": "PDF accessibility on the author or project or institution web page (true/false)",
+        "PDF on the authors' webpage / institution (boolean)": true,
+        "PDF URL": "https://arxiv.org/pdf/1904.11229",
```

Comment:
Please put here the url of the pdf hosted on the authors webpage or institution.

Reply:
Agree.. I hesitated a lot.. Actually, the PDF is only available on arxiv and many steps in the script will consider the `PDF URL` key to fetch the paper. Leaving it empty would not be great (technically) so I duplicated it (not great).

(there are some papers on the DB with an HAL or arxiv URL in that field).

Let's chat about that

Reply:
Let me check the scripts...

Reply:
Okay.. so...  
The only issue is in the `fetchData.py`: if the PDF URL is empty, no thumbnails are generated. 
In that case, we cannot rely on `Arxiv/OAI page URL` because it is supposed to be a link to the page, not the PDF on arxiv/HAL

What do you think?

Reply:
I would be in favour of having "PDF URL" to be a link to the PDF (whatever it is from hal/arxiv or institution or research gate) 

The difference arxiv vs. homepage is already handled by the booleans

This would mean a slight update of the template.

Reply:
Sorry for the late reply.
Yes we can indeed consider separately the pdf link and the webpage/OA flag.

Shall we path all the existing files, y’or just update the template ?

Reply:
No problemo ;)

I’ll do an extra PR tomorrow with the template update. Thx